test/libyul/ewasmTranslationTests/extcodecopy.yul

```diff
@@ -4,7 +4,5 @@
 }
 // ----
 // Trace:
-//   INVALID()
 // Memory dump:
-//      0: 0000000000000000000000000000000011111111000000000000000000000000
 // Storage dump:
```

Comment:
Wonder why this test doesn't return anything.

Reply:
externalCodeCopy writes nothing to memory under address `0x120` because accessMemory returns false if memory `_size > 0xffff`. In this case `_size == 0x36577b`. This means that `sstore(0, mload(0x100))` stores zeros . Memory under address 0 is zero because calling `sstore(0, mload(0x100))` overrides it when storing the pointer address.

Reply:
In previous version `sstore(0, mload(0x100))` wasn't even called because of previous `unreachable` was reached.

Reply:
> because accessMemory returns false if memory _size > 0xffff

Ah, because the current implementation of the wasm backend only supports a single page of memory. This interpreter ensures that. 

Pushed a fix as #8060.